cli/pack/pack.go

```diff
@@ -152,7 +152,11 @@ func Run(ctx *context.Ctx) error {
 		if err != nil {
 			return err
 		}
-		ctx.Pack.ResPackagePath = filepath.Join(curDir, getPackageFullname(ctx))
+		if !checkFilename(ctx) {
```

Comment:
It is better to use the condition without `not` (if possible), it is easier to read.

Reply:
fixed

Reply:
It's no more fixed. I think adding a bit more readable form (`checkFilename(ctx) == false`) is enough.

Reply:
Or we can just switch `if` and `else` branches and remove the `not` operator. The place is not so perf critical so we should optimize the route order.

Reply:
I mean:
```
		if checkFilename(ctx) {
			ctx.Pack.ResPackagePath = filepath.Join(curDir, ctx.Pack.Filename)
		} else {
			ctx.Pack.ResPackagePath = filepath.Join(curDir, getPackageFullname(ctx))
		}
```